### ISSUE TITLE
Can't view t.co links

### ISSUE THREAD

**Issue by erahhal** [AdAway/AdAway#3289]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Adding t.co (Twitter url shortener) to allowed list does not work. Saw another issue for this which recommended adding it to the allowed list. Tried deleting all browser storage, reinstalling the browser, and rebooting the phone, no luck.  Tried t.co, *t.co, and *.t.co

### Expected Behavior

T.co links are reachable with t.co in the the allowed lis

### Steps To Reproduce

Add t.co to allowed list. View link on Twitter. Note that link  still gets blocked.

### Device Configuration

- AdAway version:  5.12.1
- Ad-blocker mode: root based
- Device: oneplus 7 pro + LineageOS + microg
- Android version:  11
- Root version: Magisk 24.3

**Comment by erahhal** [comment]
Nevermind, had AdGuard home running, and it redirected in the same way as AdAway, causing my browser to show the AdAway logo even though it was blocked by a different system. Perhaps a change could be to have AdAway use a unique redirect URL to avoid confusion.

**Comment by sadkad** [comment]
Same problem here.

Adaway version: 3.2
Root
LG G6 Fulmics Rom
Android 7
Magisk 24.3

**Comment by Hermawanskaa** [comment]
Add 2 whitelist t.co and www.t.co for every shorterner link

**Comment by kaderbava** [comment]
I can't open Twitter url links(t.co) inside Twitter. Its showing privacy errors. Also i can't open telegra.ph bcoz of AdAway. After disabling AdAway, i can open it. Please fix those.

**Comment by Qynklee** [comment]
Add global.albtls.t.co; t.co; www.t.co to whitelist. It will be fixed

**Comment by kaderbava** [comment]
> 

Thanks.. It's fixed now.
